samples/dml.js

```diff
@@ -654,16 +654,27 @@ async function insertWithCustomTimeoutAndRetrySettings(
   const UNAVAILABLE_STATUS_CODE = 14;
   const retryAndTimeoutSettings = {
     retry: {
+      // The set of error codes that will be retried.
       retryCodes: [DEADLINE_EXCEEDED_STATUS_CODE, UNAVAILABLE_STATUS_CODE],
       backoffSettings: {
         // Configure retry delay settings.
+        // The initial amount of time to wait before retrying the request.
         initialRetryDelayMillis: 500,
+        // The request won't be retried after this duration has passed.
```

Comment:
`maxRetryDelayMillis` is the max waiting time before retrying. E.g., 

```
500 // waiting time after the first timeout
750 // =500 * 1.5, waiting time after the second timeout
...
64000 // max waiting time after the Nth timeout
64000 // unchanged waiting time after the (N+1)th timeout
```

Reply:
Ah thanks a lot. That makes a lot more sense now. I updated the comment.